tests/z_api_alignment_test.c

```diff
@@ -26,7 +26,11 @@
 #include <unistd.h>
 #endif
 
+#ifdef ZENOH_PICO
 #include "zenoh-pico.h"
```

Comment:
Tests are also failing because the `ZENOH_PICO` checked before its definition in `zenoh-pico.h` include